.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
```

Comment:
I'm fine any which way so long as there is a program doing the thinking on the form of the import block for me. This preference for the newline comes from the Plone isort ruleset and me and @deiferni so far failing to turn it off.

Does not seem to work with:
```
[settings]
force_alphabetical_sort_within_sections = True
force_single_line = True
from_first = True
line_length = 200
lines_after_imports = 2
lines_between_types = 0
no_sections = True
not_skip = __init__.py
```